### PR TITLE
Adds page for when user looks for image and no match is found

### PR DESCRIPTION
**Pull request by pedromartos**
## WHY

When the user looks for an image and no match is found we should show to him that image and also give him the option to upload it if he wants.
## HOW

Created a page to where the user will be redirect if we got no match on our database for his image.
There he will see his image and a button to upload it if he wants.
# GIF

iw-image-not-found

### CODE REVIEW

File: src/app/main/not-found.controller.coffee [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      $scope.uploadNotFound = (image) ->
+        $scope.$emit 'showToastrMessage',
+          type: 'info'
+          message: 'The image will be uploaded.'
+        ImageModel
+          .uploadUrl(image)
+          .then (data) ->
+            $scope.$emit 'showToastrMessage',
+              type: 'info'
+              message: 'Bu sure to fill the data for the image.'
```
Comment (gideonaa) [inline src/app/main/not-found.controller.coffee]: @pedromartos typo: "Be\* sure to fill out\* the data.."

**Comment by gideonaa** [comment]
@pedromartos is an image_id being created in order to make this temp page? In other words, is the url going to include a new image id? I wouldn't want the user to realize that they've accidentally added the image they were trying to search.

**Comment by pedromartos** [comment]
@gideonaa no image_id will be created. When the user search for an image and no match is found it will just redirect to a `/not-found` page.

**Comment by pedromartos** [comment]
Test for slack integration

**Pull request merged**